cardano-node/src/Cardano/Node/Configuration/TopologyP2P.hs

```diff
@@ -137,6 +142,7 @@ instance FromJSON LocalRootPeersGroup where
                   <$> parseJSON (Object o)
                   <*> pure hv
                   <*> o .:? "warmValency" .!= WarmValency v
+                  <*> o .: "peerTrustable"
```

Comment:
make this an optional field for backwards compatibility. Default to not trustable